scripts/run_docker_build.sh

```diff
@@ -20,16 +20,17 @@ CONDARC
 cat << EOF | docker run -i \
                         -v ${REPO_ROOT}:/staged-recipes \
```

Comment:
As I commented on https://github.com/conda-forge/conda-smithy/pull/296#discussion_r79698312 I don't think it is necessary.
But if you feel uncomfortable with that, we could change to somewhere in ```/opt```